profile/README.md

```diff
@@ -43,3 +45,8 @@ _NOTE: Please refer to https://www.redhat.com/en/blog/confidential-computing-use
 - [Full Disk Encryption](https://github.com/cc-api/full-disk-encryption)
 - [Confidential Cloud Native AI Pipeline](https://github.com/intel/cloud-native-ai-pipeline/blob/main/docs/How_to_Protect_AI_Models_in_Cloud_Native_Environments.md)
 - [Confidential Cluster](https://github.com/cc-api/confidential-cluster)
+
+# 4. Community
+
+- 2024/05/07 [CCC Attestation SIG] Q&A Update f ([Video](https://www.youtube.com/watch?v=qoAOXZpdz38), Slide)
```

Comment:
Is the extra "f" a typo?

Reply:
sorry for typo, fixed at https://github.com/cc-api/.github/pull/1/commits/9818a2b53f0e1cb9881f173b4d99235af5fcfec6